(DOCSP-28989) React Native IA update (#2746)
mongodb/docs-realm: source/sdk/react-native/manage-users.txt

   Manage Email/Password Users </sdk/react-native/manage-users/manage-email-password-users>
   Multi-User Applications </sdk/react-native/manage-users/multi-user-applications>
   Link User Identities </sdk/react-native/manage-users/link-user-identities>
   Create & Manage User API Keys </sdk/react-native/manage-users/manage-user-api-keys>

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 2
   :class: singlecol

When you use Atlas App Services to back your client app, you get access to a

> :ref:`user object <user-objects>`. Use Swift SDK methods with this user 
> object to conveniently:

Reviewer: Oops, copypasta error. (I wonder if you got this from the Node.js page, in which case, sorry! I noticed this error when I did the Java updates but fixed it in Node when I made the Java PR. But I don't know if there's a good analog for "Swift SDK methods" in RN - react hooks? Or just genericize like so?)

```suggestion
:ref:`user object <user-objects>`. With this user object,
you can:
```

Author: Shoot. I thought I fixed this. Maybe I did in a different spot. Thanks for catching!
